<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1173</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Нейросеть Хопфилда</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblLoadedImage">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>58</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblClearedImage">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>40</y>
      <width>58</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTeach">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>510</y>
      <width>87</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Обучить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClear">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>510</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Отфильтровать</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1173</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
